tests/upgrade_path/test_upgrade_path.py

```diff
@@ -84,6 +85,8 @@ def test_upgrade_path(localhost, duthosts, ptfhost, rand_one_dut_hostname,
             reboot(duthost, localhost)
             check_sonic_version(duthost, target_version)
 
+            setup_ferret(duthost, ptfhost, tbinfo)
```

Comment:
This step starts ferret. This change is missing the step to stop/cleanup ferret.
At the end of the test the ferret server on ptf also needs to be stopped, and the related files cleaned up from PTF.